x-pack/plugins/apm/public/components/shared/ManagedTable/index.tsx

```diff
@@ -11,19 +11,19 @@ import { idx } from 'x-pack/plugins/apm/common/idx';
 import { StringMap } from '../../../../typings/common';
 
 // TODO: this should really be imported from EUI
-export interface ITableColumn {
+export interface ITableColumn<T> {
   field: string;
   name: string;
   dataType?: string;
   align?: string;
   width?: string;
   sortable?: boolean;
-  render?: (value: any, item?: any) => unknown;
+  render?: (value: any, item: T) => unknown;
```

Comment:
I REALLY want `value: any` to be something like `value: T[typeof ITableColumn[field]` but don't know how to get that. In other words: how to get the type of the generic interfaces property that has the same name as the value `field`.